excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/util/mapper/SheetName.java

```diff
@@ -28,7 +28,8 @@
     AUTHORISATION_CASE_STATE("AuthorisationCaseState"),
     SEARCH_ALIAS("SearchAlias"),
     BANNER("Banner"),
-    CHALLENGE_QUESTION_TAB("ChallengeQuestion");
+    CHALLENGE_QUESTION_TAB("ChallengeQuestion"),
+    NOC_CONFIG("NoticeOfChangeConfig");
```

Comment:
@MSancaktutar NOC_CONFIG("NoticeOfChangeConfig") should be removed I believe. @kiran-yenigala-hmcts Can confirm.

Reply:
yes that column should be removed

Reply:
Well, there are code sections referring to this. There needs to be a wider range of fixes to get this branch fully correct. Current develop branch is not even compiling.